### ISSUE TITLE
Remediate homepage carousel - turn off animation and add image alt text

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#191]
1. Turn off the animation (auto rotation). Based on WCAG 2.1 guidelines, if the carousel automatically rotates, it also:

a. must have a button for stopping and restarting rotation. This is particularly important for supporting assistive technologies operating in a mode that does not move either keyboard focus or the mouse.
b. must stop rotating when keyboard focus enters the carousel, and not restart unless the user explicitly requests it to do so.
c. must stop rotating whenever the mouse is hovering over the carousel, and not restart unless the user explicitly requests it do so.

The Bootstrap 3.4 carousel component used on the homepage meets c, but does not meeting a or b. The easiest way to make it meet these criteria is to just turn off auto-rotation.

2. Add image alt text.
